nile-frontend/src/components/ProductListing/SingleProductListing.js

```diff
@@ -87,10 +87,10 @@ function SingleProductListing() {
                     <small>Ships From </small>
                     <strong>Nile</strong>
                 </p>
-                {/* <p className="ship">
+                <p className="ship">
```

Comment:
May need to have this commented out, I know it has caused issues before

Reply:
I see, Let me fix that now